src/Stepper/StepLabel.js

```diff
@@ -78,7 +85,7 @@ function StepLabel(props) {
           />
         </span>
       )}
-      <span>
+      <span className={classes.labelContainer}>
```

Comment:
I have been wondering about removing this span when `optional` isn't provided.
But didn't changed anything as I thought the extra span was worth not increasing the bundle size.
To be challenged.